.github/workflows/main.yml

```diff
@@ -1,6 +1,6 @@
 name: Main CI workflow
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Running this workflow on `push` + `pull_request` causes duplicated runs when members of the repo `push` to a `pull_request`.